packages/graphql-language-service-interface/src/fileUtils.ts

```diff
@@ -0,0 +1,57 @@
+function getExtension (pathParts: string[]): string | null {
+  return pathParts.pop() || null
```

Comment:
Should this really mutate the argument?

```suggestion
  return pathParts.length > 0 ? pathParts[pathParts.length - 1] : null
```

Reply:
ah yes, convention over terseness here.